downgrade

```diff
@@ -127,14 +135,19 @@ sort_packages() {
 output_package() {
   local pkg indicator=" "
 
-  for pkg in "${installed[@]}"; do
-    case "$2" in
-      *$pkg*)
-        indicator="*"
-        break
-        ;;
+  if [[ -n "$current" ]] && [[ "$2" == *"$current"* ]]; then
+    # Currently installed
+    indicator="+"
+  else
+    for pkg in "${installed[@]}"; do
+      case "$2" in
+        *$pkg*)
+          indicator="-"
+          break
+          ;;
     esac
```

Comment:
I think this needs to be shifted two spaces, to align with `case` above.

Reply:
Yeah I somehow overlooked this line, sorry about it.